python/tank/bootstrap/configuration.py

```diff
@@ -25,7 +25,10 @@ class Configuration(object):
     An abstraction representation around a toolkit configuration.
     """
 
-    (LOCAL_CFG_UP_TO_DATE, LOCAL_CFG_MISSING, LOCAL_CFG_DIFFERENT, LOCAL_CFG_INVALID) = range(4)
+    LOCAL_CFG_UP_TO_DATE = "LOCAL_CFG_UP_TO_DATE"
```

Comment:
numbers make everything harder to debug and test, so I switched these values to string. They are part of the internal API so no need to worry about it.

Reply:
👍